docs-ref-conceptual/msal-based-azure-cli.md

```diff
@@ -25,9 +25,14 @@ Previous versions of Azure CLI save ADAL tokens and service principal entries to
 
 The MSAL token cache and service principal entries are saved as encrypted files on Windows, and plaintext files on Linux and MacOS.
 
+> [!IMPORTANT]
+> When using Azure CLI in a pipeline, like as Azure DevOps, ensure all tasks/stages are using
+> versions of Azure CLI above v2.30.0 for MSAL-based azure CLI. Azure CLI 2.30.0 is not
+> backwards compatible with Prior versions and throws and error when using incompatible errors.
+
 ## Alternatives to consider
 
-Below are several alternatives you may consider:
+Below are a couple alternatives you may consider for backwards compatibility:
```

Comment:
(The words are [backwards compatibility](https://en.wikipedia.org/wiki/Backward_compatibility), no 's' :) )

Actually, `az account get-access-token` is not for backwards compatibility - it is a public interface and that is guaranteed to be stable.

Reply:
```suggestion
Below are a couple alternatives you may consider for backward compatibility:
```